build.sh

```diff
@@ -25,7 +25,13 @@ __scriptpath=$(cd "$(dirname "$0")"; pwd -P)
 
 "$__scriptpath/build-native.sh" $*
 if [ $? -ne 0 ];then
-   exit 1
+    exit 1
+fi
+
+# Building CoreFx.Tools before calling build-managed.sh to ensure our tools are available before running the main build.
+"$__scriptpath/Tools/msbuild.sh" "$__scriptpath/src/Tools/CoreFx.Tools/CoreFx.Tools.csproj" /v:m /m
```

Comment:
Just curious, Did we end up figuring out what changed on 2.0 in the last month that regressed this?
Also, don't we need to do this as well on build.cmd?